certbot_route53/authenticator.py

```diff
@@ -58,31 +58,32 @@ def perform(self, achalls):  # pylint: disable=missing-docstring
             return [achall.response(achall.account_key) for achall in achalls]
         except NoCredentialsError:
             raise Exception("No AWS Route53 credentials found. " + INSTRUCTIONS)
+        except ClientError as e:
+            raise Exception(str(e) + "\n" + INSTRUCTIONS)
```

Comment:
I'd rather not lose the traceback. I'd handle `ClientError` and `NoCredentialsError` the same way, as we're just appending a string to the message in both cases.

Unfortunately, this is a little kludgy to implement across  Python versions without messing the traceback nor including external libs. I'd suggest something like:
```
except (NoCredentialsError, ClientError) as e:
    e.args = ("\n".join([str(e), INSTRUCTIONS]),)
    raise
```

Proof of concept: https://gist.github.com/pkoch/0149ab65f9d18ddc6bf2dfab05c1446e